### ISSUE TITLE
Volt Gun Attachment is doing OOE Damage and proc rate seems off

### ISSUE THREAD

**Issue by E5una** [EdenServer/community#5252]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Unfortunately I cannot find concrete data about this attachments in era other than a video showing the damage, but we seem to have the OOE updated version that abolished a level correction aspect. 

JPwiki which describes the single digit damage and mentions how the 2011 change affected things: http://wiki.ffo.jp/html/8752.html

FFXIClopedia page that also mentions the 2011 change: https://ffxiclopedia.fandom.com/wiki/Volt_Gun

Era FFXIClopedia page detailing that it dealt A LOT of damage to too weak mobs. This missing piece of data is why I hadn't made this bug report yet, but hopefully someone can find evidence and add to it so we can ballpark it.

https://ffxiclopedia.fandom.com/wiki/Volt_Gun?oldid=580089

**These two sources are conflicting about if MAB has an effect on damage**

Here is a 2010 video of a L58 PUP (he dings 59 near the end of the video) using this attachment. The first proc is with 0 thunder maneuvers active: for 9 damage:

![image](https://user-images.githubusercontent.com/13668758/189126301-acb285a6-1439-41bf-92a0-e3a7132d220c.png)

Second proc is 8 damage with no thunder manevuers:
![image](https://user-images.githubusercontent.com/13668758/189126936-ff2229f5-243d-443d-ac55-ff52763efa40.png)

Then 8 damage, no thunder:
![image](https://user-images.githubusercontent.com/13668758/189135290-78cf8ca3-d75e-4184-b134-55d0731ead5a.png)

Then 10 damage, no thunder maneuver:
![image](https://user-images.githubusercontent.com/13668758/189127091-60afb797-80e4-4183-bc12-366a751166e5.png)

Then 10 again, no thunder:
![image](https://user-images.githubusercontent.com/13668758/189129708-d7612825-c931-42fb-aaf4-9f343a215210.png)

8, no thunder:
![image](https://user-images.githubusercontent.com/13668758/189131992-e26659c2-4ebf-43ba-9351-f0cfe3bb7679.png)

10, no thunder:
![image](https://user-images.githubusercontent.com/13668758/189132092-7cc3b459-a084-4b02-b81e-6e8fb0bbdbb4.png)

9, no thunder:
![image](https://user-images.githubusercontent.com/13668758/189132363-323c7ae5-0956-4166-8034-aeee6863ca0b.png)

10, no thunder:
![image](https://user-images.githubusercontent.com/13668758/189132535-a6e96b37-fc67-47e6-bb3f-88657a7ceee2.png)

There are more procs, but rather than post all the screenshots, I'll upload an excel sheet with all of the data: https://docs.google.com/spreadsheets/d/13vTkUUrgKhcePiyIn63z5EvL2JIStapd1cY5vI_bntk/edit?usp=sharing

Average damage (done by hand because I didn't want to put 0s in the additional effect column or learn how to do this without 0s) is 8.347826.

Performing the same test on Eden, my results are also in the spreadsheet. I only had Volt Gun on and had no maneuvers active:

![image](https://user-images.githubusercontent.com/13668758/189143401-90ac8ec3-6f15-4436-b20a-2b827041005b.png)
![image](https://user-images.githubusercontent.com/13668758/189143502-3d12de9f-9462-4b3b-91e3-58c0815a0eef.png)
![image](https://user-images.githubusercontent.com/13668758/189143542-4380b740-a216-4c5b-bd64-b6d800148300.png)
![image](https://user-images.githubusercontent.com/13668758/189143584-bfa00dbd-7996-4851-98ca-0b9ca8c7281f.png)
![image](https://user-images.githubusercontent.com/13668758/189143695-2d45339f-91fe-425f-bf8b-22351e7afb6b.png)

The summary of the results are below:

![image](https://user-images.githubusercontent.com/13668758/189143774-a2d5c4ae-c1e4-4059-949f-b1a3e9930633.png)

I realize this sample size is too small to draw any meaningful conclusions, but its the best I can find and have been looking on and off for months. 

Unfortunately I can't find a video of a PUP fighting something higher level in era with Volt Gun on, or drastically lower level, since both of those would tell a much more important story (we expect A LOT of damage on low level, and no damage on high level).

On Eden we don't have the level correction at all though, which is post 2011. Here is the summary of me testing against a level 8ish hornet (I didn't check its level, but it was outside Dangruf Wadi) and a Greater Colibri. Both tests were performed with 0 maneuvers active, only Volt Gun equipped, and as a level 58 PUP. I capped my accuracy for the Greater Colibri test and used !Immortal in both tests.

![image](https://user-images.githubusercontent.com/13668758/189150292-c5f24b8e-fd4d-4136-9b8a-1ddb20a0dc57.png)

In summary: Volt Gun isn't proccing often enough, but is dealing too much damage to DC-IT+ mobs. Source on the internet indicate that it should deal more to Too Weak / EP, but I can't find data to get numbers for that. Hoping someone else can help provide this information. We also need to see it proc vs VT or IT+ mobs.

Version 30181205_0
